flink/v1.15/flink/src/test/java/org/apache/iceberg/flink/RowDataConverter.java

```diff
@@ -62,22 +62,8 @@ private static RowData convert(Types.StructType struct, Record record) {
     List<Types.NestedField> fields = struct.fields();
     for (int i = 0; i < fields.size(); i += 1) {
       Types.NestedField field = fields.get(i);
-
```

Comment:
Please revert this unnecessary whitespace change.

Reply:
I have reverted the unnecessary whitespace change.

Reply:
@rdblue anything do I need to modify?